ls1intum/Artemis
# `Development`: Drop `lti_user_id` table

## Do not deploy to test server

### Checklist
#### General

### Motivation and Context
 
Continuation of https://github.com/ls1intum/Artemis/pull/5857. The table drop is in a separate/follow-up PR so that the previous PR could be deployed to test servers without issues.

### Description
Drops the now unused and unnecessary **lti_user_id** table.

### Steps for Testing

- Database access in the test server

1. Verify the PR was deployed to the test server
2. Verify the table was dropped

### Review Progress

#### Code Review
#### Manual Tests
